src/app/components/preview-card/preview-card.component.scss

```diff
@@ -0,0 +1,60 @@
+.mat-card {
+  position: relative;
+  margin: 20px 10px;
+  transition: 0.2s;
+  transition-property: box-shadow;
+
+  &:hover, &:focus {
+    box-shadow: 0 5px 11px 0 rgba(0, 0, 0, 0.18),
+      0 4px 15px 0 rgba(0, 0, 0, 0.15);
+  }
+}
+
+.mat-card-title {
+  margin-left: -16px; // align header tile with content (mat-card-text not responsible)
+}
+
+.tags-box {
+  display: inline-table;
+}
+
+.tags {
+  background-color: #e55a17; // #8e92de
+  color: #fff;
+  border-radius: 20px;
+  margin: 8px 8px 8px 0;
+}
+
+.info {
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  grid-column-gap: 2em;
+  align-items: center;
+
+  .intros {
+    justify-self: left;
+  }
+
+  .dates {
+    justify-self: right;
+    text-align: right;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    b {
+      text-align: right;
+      padding-right: 10px;
+    }
+  }
+}
+
+.synapse-icon {
```

Comment:
Rename to something less platform-specific, like `platform-icon` or maybe just `logo`, since not all challenges will be from Synapse.

Reply:
@rrchai to address this comment

Reply:
I have changed the method to get images in 87a4370. Now, it will call images based on the platform information in challenge data (since we only have synapse now, I just use const variable `platform = 'synapse'`)